tests/e2e/specs/01Navbar.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { getElem } from './utilities/utils';
+
+import { topNavbar } from './utilities/01Navigation.selectors';
+
+describe('01 Navbar', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+    it('Navbar overview', () => {
+        getElem(topNavbar).should('be.visible');
+    });
+});
```

Comment:
@Andresws12 Here, I think that the sidebar is no added because is WIP really, but I think that to can test better the navbar, you can add the button of translations in the test.

Reply:
@gonzalo-plaza This spect is erased in the future, now he is here as an example